src/client-private.ts

```diff
@@ -21,7 +21,7 @@ export async function setupStreamingClient (this: GenesysCloudWebrtcSdk): Promis
     host: this._config.wsHost || `wss://streaming.${this._config.environment}`,
     apiHost: this._config.environment,
     logger: this.logger,
-    appName: 'sdk-demo',
+    appName: 'webrtc-sdk',
```

Comment:
I didn't mean to change it to `sdk-demo` in `master`. But hopefully, `webrtc-sdk` isn't a breaking change. I can revert this if it is (I don't remember what the timeline parser is looking for off the top of my head).